src/render/md.jl

```diff
@@ -1,35 +1,48 @@
-## Docs-specific rendering
+type EntriesMD
+    include_internal::Bool
+    exported::Dict{Symbol, Vector{(Module, Any, Entry, ByteString)}}
+    internal::Dict{Symbol, Vector{(Module, Any, Entry, ByteString)}}
+
+    EntriesMD(config::Config) = new(config.include_internal,
+            Dict{Symbol, Vector{(Module, Any, Entry, ByteString)}}
+                        ([(c, []) for c in config.category_order]),
+            Dict{Symbol, Vector{(Module, Any, Entry, ByteString)}}
+                        ([(c, []) for c in config.category_order])
+            )
+end
```

Comment:
Type-inference should do fine guessing the necessary types in the constructor. Try replacing the `Dict{Symbol, Vector{(Module, Any, Entry, ByteString)}}` with just `Dict`.